test_scripts/Polices/build_options/TC002_SDL_Build_HTTP_Flow_DEXTENDED_POLICY_No_Flag.lua

```diff
@@ -0,0 +1,58 @@
+---------------------------------------------------------------------------------------------
+-- Requirements summary: 
+--     [PolicyTableUpdate] Support of "http" flow of Policy Table Update
+--
+-- Description: 
+-- SDL should be successfully built "-DEXTENDED_POLICY:HTTP" flag is empty
+-- 1. Performed steps
+-- Build SDL 
+--
+-- Expected result:
+-- SDL is successfully built
+-- The flag -DEXTENDED_POLICY:HTTP has no value
+-- PTU passes successfully 
+
+---------------------------------------------------------------------------------------------
+
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+
+--[[ Required Shared libraries ]]
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+--local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonPreconditions = require ('user_modules/shared_testcases/commonPreconditions')
+local testCasesForPolicyTable = require('user_modules/shared_testcases/testCasesForPolicyTable')
+local testCasesForBuildingSDLPolicyFlag = require('user_modules/shared_testcases/testCasesForBuildingSDLPolicyFlag')
+  
+--[[ General Precondition before ATF start ]]
+-- commonFunctions:SDLForceStop()
+testCasesForBuildingSDLPolicyFlag:Update_PolicyFlag("ENABLE_HTTP_POLICY")
+testCasesForBuildingSDLPolicyFlag:CheckPolicyFlagAfterBuild("ENABLE_HTTP_POLICY")
```

Comment:
@maria-mih The call of function should be
testCasesForBuildingSDLPolicyFlag:CheckPolicyFlagAfterBuild("EXTENDED_POLICY","")